src/Listener/ViewListener.php

```diff
@@ -467,7 +466,7 @@ protected function _getAllowedActions()
     {
         $actions = $this->_action()->getConfig('scaffold.actions');
         if ($actions === null) {
-            $actions = $this->_crud()->getConfig('actions');
+            $actions = array_keys($this->_crud()->getConfig('actions'));
```

Comment:
There's an off chance someone might be stuffing link config too into action class config, they will have to update their code to use `scaffold.actions` instead.